docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -14,41 +14,42 @@ The server node is usually set up to support [multiple private states](../../con
 
 Configure the qlight client using the following command line options:
 
-- `--qlight.client`: This marks the node as a qlight client.
-- `--qlight.client.psi`: This specifies the PSI which this client will support (default = "private").
-- `--qlight.client.serverNode`: The Node ID of the server node.
-- `--qlight.client.serverNodeRPC`: The RPC URL of the server node.
+- [`--qlight.client`](../../reference/cli-syntax.md#qlightclient): This marks the node as a qlight client.
+- [`--qlight.client.psi`](../../reference/cli-syntax.md#qlightclientpsi): This specifies the PSI which this client will support (The default is `private`).
+- [`--qlight.client.serverNode`](../../reference/cli-syntax.md#qlightclientservernode): The Node ID of the server node.
+- [`--qlight.client.serverNodeRPC`](../../reference/cli-syntax.md#qlightclientservernoderpc): The RPC URL of the server node.
 
 If the server node has the RPC API secured using TLS, then the qlight client requires the following:
```

Comment:
Since the options descriptions are simply repeated here, I recommend changing to:

```suggestion
If the server node has the RPC API secured using TLS, then you must set the [`--qlight.client.rpc.tls.*`](../../reference/cli-syntax.md#qlightclientrpctls) command line options.
```

And deleting the bulleted list.